pkgs/pkgs-lib/default.nix

```diff
@@ -0,0 +1,11 @@
+# pkgs-lib is for things that should be in lib, but can't beceause
```

Comment:
```suggestion
# pkgs-lib is for functions that can't be in lib, because
```

Reply:
Done with adjustments in https://github.com/NixOS/nixpkgs/compare/76e8e4a26fa451bbc1832e0cd599080795227593..83b16885f526d6ab7b39e98159e2b48024f3238c#diff-fa126a2182114f76a8e8df7ed256395d